ydb/docs/en/core/integrations/liquibase.md

```diff
@@ -0,0 +1,551 @@
+| `Float`, `Double`, `Decimal(22, 9)`     | A real number                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `Interval`                              | [Format](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html#parse(java.lang.CharSequence)) `java.time.Duration`, prefix `PT` is optional                                                                                                                                                                                                                                                                                       |
+| `Date`, `Datetime`, `Timestamp`         | Format [ISO-8601](https://ru.wikipedia.org/wiki/ISO_8601), in Java, the following format is used: [ISO_LOCAL_DATE](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE), [ISO_LOCAL_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME), [ISO_INSTANT](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_INSTANT) |
+
+{% note warning %}
+
+To understand which SQL statements {{ydb-short-name}} can perform, read the documentation for the query language [YQL](https://ydb.tech/docs/en/yql/reference/).
+
+{% endnote %}
+
+{% note info %}
+
+It is important to note that custom YQL instructions can be applied via [native SQL queries](https://docs.liquibase.com/concepts/changelogs/sql-format.html).
+
+{% endnote %}
+
+## How to use it? {#using}
+
+There are two ways:
+
+{% list tabs %}
+
+- Programmatically from Java / Kotlin applications
+
+  The project's [README](https://github.com/ydb-platform/ydb-java-dialects/tree/main/liquibase-dialect) describes how to use it from Java or Kotlin in detail. There is also an [example of a Spring Boot application](https://github.com/ydb-platform/ydb-java-examples/tree/master/jdbc/spring-liquibase-app) using it.
+
+- Liquibase CLI
+
+  First, you need to install Liquibase itself using [one of the recommended methods](https://docs.liquibase.com/start/install/home.html). Then you need to place the `.jar` archives of [{{ ydb-short-name }} JDBC driver](https://github.com/ydb-platform/ydb-jdbc-driver/releases) and Liquibase [{{ ydb-short-name }} dialect](https://mvnrepository.com/artifact/tech.ydb.dialects/liquibase-ydb-dialect/1.0.0) into the `internal/lib` folder.
```

Comment:
btw isn't the official maven repo this one https://repo.maven.apache.org/maven2/tech/ydb/ ? and the referenced one seems to be some unofficial one